taker-frontend/src/App.tsx

```diff
@@ -79,6 +74,18 @@ export const App = () => {
         void fetchDaemonVersion(setDaemonVersion);
     }, []);
 
+    const {
+        isOpen: outdatedWarningIsVisible,
+        onOpen: onOutdatedWarningOpen,
+        onClose: onOutdatedWarningClose,
+    } = useDisclosure();
+
+    useEffect(() => {
+        if (githubVersion && daemonVersion) {
+            onOutdatedWarningOpen();
```

Comment:
Does this mean it's always open? :)

Reply:
No, because it only reacts on state changes of `[githubVersion, daemonVersion, onOutdatedWarningOpen]`, so if the user closes it and the state does not change then it will be closed.